src/app/components/progress-table/progress-table.component.html

```diff
@@ -170,14 +171,39 @@
         </ion-button>
       </ng-template>
     </ngx-datatable-column>
-  </ngx-datatable>
 
-  <ion-button color="primary" fill="clear" (click)="switchType()">
-    <ng-container *ngIf="type === 'team'">
-      VIEW PROGRESS BY STUDENT
-    </ng-container>
-    <ng-container *ngIf="type === 'student'">
-      VIEW PROGRESS BY PROJECT
-    </ng-container>
-  </ion-button>
+    <ngx-datatable-footer style="position: fixed; bottom: 0px;" *ngIf="true">
+      <ng-template
+        ngx-datatable-footer-template
+        let-rowCount="rowCount"
+        let-pageSize="pageSize"
+        let-selectedCount="selectedCount"
+        let-curPage="curPage"
+        let-offset="offset">
+        <ion-button class="footer-content" color="primary" fill="clear" (click)="switchType()">
+          <ng-container *ngIf="type === 'team'">
+            VIEW PROGRESS BY STUDENT
+          </ng-container>
+          <ng-container *ngIf="type === 'student'">
+            VIEW PROGRESS BY PROJECT
+          </ng-container>
+        </ion-button>
+        <div class="count footer-content">
+          {{ rowCount }} Total
+        </div>
+        <datatable-pager
+          [pagerLeftArrowIcon]="'datatable-icon-left'"
+          [pagerRightArrowIcon]="'datatable-icon-right'"
+          [pagerPreviousIcon]="'datatable-icon-prev'"
+          [pagerNextIcon]="'datatable-icon-skip'"
+          [page]="curPage"
+          [size]="pageSize"
+          [count]="rowCount"
+          [hidden]="!((rowCount / pageSize) > 1)"
```

Comment:
This should already be a built in logic that we don't need to specify explicitly

Reply:
we can remove it but after we remove pagination will show if there have only one page available.
without this pagination works but it will show if there only one page have. because of we added pagination separately we need to specify every thing what we need to show.